CONTRIBUTING.md

```diff
@@ -271,7 +271,7 @@ When in prerelease mode, the automatic PR release process will no longer release
 1. Run `pnpm exec release` to publish your new release.
 1. Run `git push && git push --tags` to push your new release to GitHub.
 1. Run `git push release/0.X:latest` to push your release branch to `latest`.
-1. Go to https://github.com/withastro/astro/releases/new and create a new release. Copy the new changelog entry from https://github.com/withastro/astro/blob/latest/packages/astro/CHANGELOG.md.
+1. Go to <https://github.com/withastro/astro/releases/new> and create a new release. Copy the new changelog entry from <https://github.com/withastro/astro/blob/latest/packages/astro/CHANGELOG.md>.
```

Comment:
I think GH markdown should work without the angle brackets, so I'm not sure if this needs to be added here.

Reply:
While true that it is not required in the Github flavoured markdown, I would opt to add them, we also do this everywhere else IIRC

Reply:
Interesting, I'm not sure if we're using angle brackets everywhere now, but if it's a pattern the docs prefer, I'm fine with adding it.

Reply:
It might be a good idea to spend some time looking at other documents later. For now, I resolve this conversation

Reply:
I personally haven't seen the usage of angle brackets in the docs before, we rarely expose links like this, we rather associate it with a piece of content using the `[link title](link)` syntax.

> We should also consider people viewing through other markdown viewers

I would not mind adding them, I'm just not sure where someone would read this if not inside a GitHub product (GitHub.com, GitHub Mobile, etc.) and I am a bit worried if we change this in READMEs that are parsed into our docs site, as I'm not sure if it consider this syntax.

Reply:
It seems better not to consider other viewers (with little demand) for the sake of consistency and safety. I'll remove the angle brackets.